US COVID-19.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "US COVID-19"
+author: "Xing Yanfeng"
+date: "2020/7/20"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = F)
+```
+
+This is graphs of **US COVID-19 for 2020** data to show tendency of case and death increase. Data were downloaded from [*The New York Times*](https://www.nytimes.com/interactive/2020/us/coronavirus-us-cases.html). 
+
+```{r load library}
+library(ggplot2)
+library(readr)
+```
+
+### US COVID-19 data for 2020
+
+
+```{r plotting}
+#importing data
+us_data <- read_csv("us.csv", col_types = "Dii") 
+
+#plotting data
+ggplot(us_data, aes(x = date)) +
+  geom_col(aes(y = cases), color = "#0080FF", fill = "#0080FF", alpha = 0.5) +
+  geom_col(aes(y = deaths), color = "red", fill = "red", alpha = 0.5) +
```

Comment:
Kind of a silly comment - adding the outline with a decreased alpha value was very aesthetically pleasing